contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6216,7 +6216,23 @@ bbf_ExecDropStmt(DropStmt *stmt)
 	Relation		relation = NULL;
 	Oid				schema_oid;
 	ListCell		*cell;
-	const char *logicalschema = NULL;
+	const char		*logicalschema = NULL;
+
+	/*
+	 * BABEL-4390
+	 * List of procedure names that are not allowed to be dropped. These procedures 
+	 * are considered essential or restricted due to security or operational reasons.
+	 */
+	const char		*restricted_procedures[] = {
```

Comment:
Blocking: Should we not block dropping This procedures from PSQL Endpoint as well?